### ISSUE TITLE
Medaka v1.7.3

### ISSUE THREAD

**Issue by incoherentian** [rpetit3/dragonflye#17]
Dear @rpetit3, is dragonFlye still being maintained? 

Selfish request for medaka to be updated if so! 

(I can never get medaka working independently polishing my Flye assemblies, so when I'm doing LR-only assemblies like to use dragonFlye. A bit funny since I originally used dragonFlye for quick&easy polypolish! Such a nice pipeline:)

**Comment by rpetit3** [comment]
Hi @incoherentian 

Unfortunately, Medaka is out of my hands. I see there is a PR open on Bioconda, but it needs the maintainer to review it. 

Once that is merged into Bioconda, Dragonflye should work with 1.7.3. You might have to rebuild, and/or be explicit with medaka pinning (e.g. `medaka=1.7.3` in mamba create command).

Cheers,
Robert

**Comment by incoherentian** [comment]
> 

Ah OK. I'm a little blind when it comes to those things and just load separate SLURM modules where dependencies don't sort themselves in mamba>conda (I have to do this for bcftools with dragonflye using medaka, which is probably wrong but works :'). I'll look into what you mean with pinning medaka once the newest version is up on bioconda!

Thanks so much for the super fast reply!

Supplemental question - (Was just looking at your other stuff since dragonflye is so nice, so was reading into your bactopia analytical platform.) Have you considered replacing or supplementing the suggestion to use unicycler with bactopia, with your own dragonFlye using short read polishing?

**Comment by rpetit3** [comment]
No problem! 

Oh yeah, Dragonflye was originally developed because I wanted ONT assembly support in Bactopia with a Shovill-like experience.

Currently Bactopia has these options for assembly:

1. ONT reads only
     - Use Dragonflye
2. ONT and Illumina reads
     - Dragonflye with ONT, then short polishing with Illumina reads
     - Hybrid with Unicycler
3. Illumina reads
     - Use Shovill (skesa, spades, velvet, or megahit)
     - Use Unicycler
     
I'm close to a version update on Bactopia, then the docs will need some updating!

**Comment by incoherentian** [comment]
Nice!

**Comment by incoherentian** [comment]
Hi again @rpetit3 ! Didn't realize my message on medaka would be listed here...

I finally gave up on someone else asking, posted on medaka - but their suggested solution was asking me to do... something. (I'm not certain what, so assuming insufficient competency to try!) In absence of that option I tried adapting your pinning suggestion using the current bioconda Medaka:

`mamba create -n dragonflye_m172 -c bioconda -c conda-forge dragonflye=1.0.14 medaka=1.7.2`

This yielded an error in CUDA 11.2 and 11.7 (this is all on a SLURM node via a BASH srun so switching CUDA versions is easy) :

**Encountered problems while solving:  - nothing provides __cuda needed by tensorflow-2.10.0-cuda112py310he87a039_0**

I resolved that with CUDA 11.7 up on the node and a CUDA override:

`CONDA_OVERRIDE_CUDA=11.7 mamba create -n dragonflye_m172 -c bioconda -c
conda-forge dragonflye=1.0.14 medaka=1.7.2`

I loaded that env (and bcftools 1.16.0 as a SLURM module) then assembled some basecalled R9.4.1 data I'd assembled a couple different ways recently and had handy. It used the almost-newest medaka, and at a glance it spit out the genome & plasmid expected. 

(Is that a useful thing to share? Still finding my legs on github...)

Either way, thanks again for your assembly pipeline from the suitably thematic land of winged dragons - Wales!

**Comment by rpetit3** [comment]
Awesome, glad it worked out for you. I've updated the Bioconda pull request, will hopefully be able to get 1.7.3 merged into bioconda this week

**Comment by incoherentian** [comment]
Hi again! I just mamba installed dragonflye 1.1.0 without pinning anything and found that it installed almost-newest medaka by default which is great :) 

Do you think any major changes will be required for flye 2.9.2? Wondering if that would be fairly safe to pin &/or if you've done anything with it yet!

**Comment by incoherentian** [comment]
Answered my own premature question - seems flye 2.9.2 isn't available via conda yet!
